src/api/api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export const BASE_URL = 'http://localhost:8000';
```

Comment:
This should be environment variable, so it's easy to change in case the api host changes. Create `.env` and `.env.example` file and add `.env` into `.gitignore` with proper values.

Hint: You can find working example of this in the api repository.